Scribe/MainEditorForm.cs

```diff
@@ -404,37 +406,58 @@ private void UpdateLibraryDataDisplay()
         /// <param name="e">Ignored.</param>
         private void ContentAlteredEventHandler(object sender, EventArgs e)
         {
+            // TODO Remove this debug statement.
+            MainToolStripStatusLabel.Text = ChangeManager.Count.ToString();
+
             if (sender is TextBox textbox
                 && string.Compare(textbox.Text,
                                   EditableControls[typeof(TextBox)][textbox],
                                   comparisonType: StringComparison.OrdinalIgnoreCase) != 0)
             {
-                EditableControls[typeof(TextBox)][textbox] = textbox.Text;
-                HasUnsavedChanges = true;
+                ChangeManager.AddAndExecute(new Change(EditableControls[typeof(TextBox)][textbox], textbox.Text, textbox.Name,
+                                            (object databaseValue) => { _ = databaseValue.ToString(); HasUnsavedChanges = true; },
```

Comment:
Is the DatabaseValue getting updated? It looks like you're doing nothing with the value of databaseValue. (assigning to _ does nothing, I think?)

Reply:
True, it is doing nothing here because this is a test form.  In the real form there is code here to access the appropriate value somewhere in All.SomeModelCollection.Get(someModelID);   👍